var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -33,6 +33,18 @@ class Qthreads(AutotoolsPackage):
         default=True,
         description='hwloc support'
     )
+    variant('spawn_cache',
+            default=True,
+            description='enables worker specific cache of spawns')
+    variant('scheduler', default='nemesis',
+            values=('nemesis', 'lifo', 'mutexfifo', 'mtsfifo',
+                    'sherwood', 'distrib', 'nottingham'),
+            multi=False,
+            description='Specify which scheduler policy to use')
+    variant('static', default=True, description='Build static library')
+    variant('stack_size',
+            default='4096',
```

Comment:
Can you test this to make sure it works with other values? Other packages add extra logic like verification that the value is an int.